spanner/spannertest/integration_test.go

```diff
@@ -889,6 +889,42 @@ func TestIntegration_ReadsAndQueries(t *testing.T) {
 				{int64(1), int64(25)}, // Jack(ID=1, Tenure=10), Sam(ID=3, Tenure=9), George(ID=5, Tenure=6)
 			},
 		},
+		{
```

Comment:
I think there's too many test cases here. This doesn't need to exhaustively test, just give it a quick exercise. Maybe just grab one of them from https://cloud.google.com/spanner/docs/aggregate_functions#avg.

Reply:
Sorry, I didn't understand the test policy about how exhaustively tests should be implemented. I kept only one simple test and removed the others.

Btw, I tried to use either of the sample SQLs written in the official doc, but they don't seem to work correctly. 
I raised that as another issue here.
https://github.com/googleapis/google-cloud-go/issues/3296

So as I said in the above, I kept one original test case using the prepared data in this test func for the time being.

Reply:
That's fine, but you can still use that example, just slightly modified. There's other test cases here like that. We don't want to tie too many test cases to the built-up test data, as if that ever changes then there's lots of tests that need adjusting; for something like `AVG` it should suffice to see it exercised in an array provided as a param.

Reply:
Though I added the related comment in #3296 , I cannot make the SQL work as it is in the official doc.

But,
> We don't want to tie too many test cases to the built-up test data, as if that ever changes then there's lots of tests that need adjusting; for something like AVG it should suffice to see it exercised in an array provided as a param.

I completely agree with that. Actually, I feel that when I was implementing. So, I want to somehow create the test that doesn't depend on the built-up test data.

Reply:
I tried to work out the case that can test "AVG" without the built-up data, but, unfortunately, I could not get that from the currently supported SQLs. 
Let me know if you have any.

Reply:
Sure, alright. Let's just use the existing data for this.